GeosCore/cldj_interface_mod.F90

```diff
@@ -1213,4 +1216,141 @@ FUNCTION Get_RH_Index( RH ) RESULT( index )
+    amIRoot = Input_Opt%amIRoot
+
+    fileName = TRIM( Input_Opt%CloudJ_Dir ) // 'CJ77_inp.dat'
+    CALL Check_File_For_DryRun( fileName, amIRoot )
+
+    fileName = TRIM( Input_Opt%CloudJ_Dir ) // 'FJX_spec.dat'
+    CALL Check_File_For_DryRun( fileName, amIRoot )
+
+    fileName = TRIM( Input_Opt%CloudJ_Dir ) // 'FJX_scat-aer.dat'
+    CALL Check_File_For_DryRun( fileName, amIRoot )
+
+    fileName = TRIM( Input_Opt%CloudJ_Dir ) // 'FJX_scat-cld.dat'
+    CALL Check_File_For_DryRun( fileName, amIRoot )
+
+    fileName = TRIM( Input_Opt%CloudJ_Dir ) // 'FJX_scat-ssa.dat'
+    CALL Check_File_For_DryRun( fileName, amIRoot )
+
+    fileName = TRIM( Input_Opt%CloudJ_Dir ) // 'FJX_scat-UMa.dat'
+    CALL Check_File_For_DryRun( fileName, amIRoot )
+
+    fileName = TRIM( Input_Opt%CloudJ_Dir ) // 'FJX_scat-geo.dat'
+    CALL Check_File_For_DryRun( fileName, amIRoot )
+
+    fileName = TRIM( Input_Opt%CloudJ_Dir ) // 'atmos_std.dat'
+    CALL Check_File_For_DryRun( fileName, amIRoot )
+
+    fileName = TRIM( Input_Opt%CloudJ_Dir ) // 'atmos_h2och4.dat'
+    CALL Check_File_For_DryRun( fileName, amIRoot )
+
+    fileName = TRIM( Input_Opt%CloudJ_Dir ) // 'atmos_geomip.dat'
+    CALL Check_File_For_DryRun( fileName, amIRoot )
+
+    fileName = TRIM( Input_Opt%CloudJ_Dir ) // 'FJX_j2j.dat'
+    CALL Check_File_For_DryRun( fileName, amIRoot )
+
+  END SUBROUTINE CloudJ_DryRun_Check
+!EOC
+!------------------------------------------------------------------------------
+!                  GEOS-Chem Global Chemical Transport Model                  !
+!------------------------------------------------------------------------------
+!BOP
+!
+! !IROUTINE: Check_File_For_DryRun
+!
+! !DESCRIPTION: Checks the status of a single file in dry-run simulations.
+!\\
+!\\
+! !INTERFACE:
+!
+ SUBROUTINE Check_File_for_DryRun( fileName, amIRoot )
+!
+! USES:
+!
+    USE ErrCode_Mod
+    USE Input_Opt_Mod, ONLY : OptInput
+!
+! !INPUT PARAMETERS:
+!
+    CHARACTER(LEN=*), INTENT(IN) :: fileName   ! Name of file to check
+    LOGICAL,          INTENT(IN) :: amIRoot    ! Are we on the root core?
+!
+! !REVISION HISTORY:
+!  19 Dec 2023 - R. Yantosca - Initial version
+!  See https://github.com/geoschem/geos-chem for complete history
+!EOP
+!------------------------------------------------------------------------------
+!BOC
+    
+    ! Scalars
+    LOGICAL            :: fileExists
+
+    ! Strings
+    CHARACTER(LEN=512) :: fileMsg
+
+    !========================================================================
+    ! Check_File_for_DryRun begins here!
+    !========================================================================
+
+    ! Test if the file exists
+    INQUIRE( FILE=TRIM( fileName ), EXIST=fileExists )
+
+    ! Test if the file exists and define an output string
+    IF ( fileExists ) THEN
+       fileMsg = 'CLOUD-J: Opening'
+    ELSE
+       fileMsg = 'CLOUD-J: REQUIRED FILE NOT FOUND'
```

Comment:
Same comment as above. I would stay away from labeling non-Cloud-J calls as CLOUD-J.

Reply:
Now updated in 22c8094